README.md

```diff
@@ -8,8 +10,75 @@
 
 ***Due to limitations imposed by Github and occasional slow server response times, some badges may require a page refresh to load.**
 
-# README
+## Office of Family Assistance (OFA) & Temporary Assistance for Needy Families (TANF) Data Reporting System.
+
+Welcome to the home of the TANF Data Portal (TDP).
+
+## What We're Building and Why
+
+- [Product planning page]( https://github.com/HHS/TANF-app/blob/main/docs/README.md) includes latest information on our product mission, goals, roadmap, and backlog. 
+
+## Getting Started
+
+### Running the Application locally
+
+Both the frontend (`http://localhost:3000`) and the backend (`http://localhost:8080`) applications run within Docker.  Instructions for running these containers are below:
+
+```
+$ cd tdrs-frontend && docker-compose up -d
+$ cd tdrs-backend && docker-compose up -d 
+```
+
+After the above commands there will be a total of 5 running containers
+
+```
+CONTAINER ID        IMAGE                        COMMAND                  CREATED             STATUS                            PORTS                    NAMES
+c803336c1f61        tdp                          "bash -c 'python wai…"   3 seconds ago       Up 3 seconds                      0.0.0.0:8080->8080/tcp   tdrs-backend_web_1
+20912a347e00        postgres:11.6                "docker-entrypoint.s…"   4 seconds ago       Up 3 seconds                      5432/tcp                 tdrs-backend_postgres_1
+9c3e6c2a88b0        owasp/zap2docker-weekly      "sleep 3600"             4 seconds ago       Up 3 seconds (health: starting)                            tdrs-backend_zaproxy_1
+7fa018dc68d1        owasp/zap2docker-stable      "sleep 3600"             41 seconds ago      Up 40 seconds (unhealthy)         0.0.0.0:8090->8090/tcp   zap-scan
+63f6da197629        tdrs-frontend_tdp-frontend   "/docker-entrypoint.…"   41 seconds ago      Up 40 seconds                     0.0.0.0:3000->80/tcp     tdp-ui
+```
+
+Below is a GIF of both the frontend and backend running locally
+
+![GIF of frontend and backend running locally](https://user-images.githubusercontent.com/44377678/104548466-e1022380-55fe-11eb-9a7b-eea7cda395d4.gif)
+
+Detailed instructions for running unit and end-to-end integration testing on frontend and backend are available below
+
+- [Frontend](https://github.com/HHS/TANF-app/tree/main/tdrs-frontend)
+- [Backend](https://github.com/HHS/TANF-app/tree/main/tdrs-backend)
+
+
+## Infrastructure
+
+TDP Uses Infrastructure as Code (IaC) and DevSecOps automation
```

Comment:
I might lean towards "show" rather that "tell" here but that's not a requirement -- feel free to keep, remove, or change this line as you see fit!